File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/NUR2016U02GrdListInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    [Serializable]
    public class NUR2016U02GrdListInfo
    {
        private String _select;
        private String _hospCodeLink;
        private String _yoyangName;
        private String _bunhoLink;
        private String _suname;
        private String _address1;
        private String _birth;

        public String Select
        {
            get { return this._select; }
            set { this._select = value; }
        }

        public String HospCodeLink
        {
            get { return this._hospCodeLink; }
            set { this._hospCodeLink = value; }
        }

        public String YoyangName
        {
            get { return this._yoyangName; }
            set { this._yoyangName = value; }
        }

        public String BunhoLink
        {
            get { return this._bunhoLink; }
            set { this._bunhoLink = value; }
        }

        public String Suname
        {
            get { return this._suname; }
            set { this._suname = value; }
        }

        public String Address1
        {
            get { return this._address1; }
            set { this._address1 = value; }
        }

        public String Birth
        {
            get { return this._birth; }
            set { this._birth = value; }
        }

        public NUR2016U02GrdListInfo() { }

        public NUR2016U02GrdListInfo(String select, String hospCodeLink, String yoyangName, String bunhoLink, String suname, String address1, String birth)
        {
            this._select = select;
            this._hospCodeLink = hospCodeLink;
            this._yoyangName = yoyangName;
            this._bunhoLink = bunhoLink;
            this._suname = suname;
            this._address1 = address1;
            this._birth = birth;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/NUR2016U02GrdListInfo.java
package nta.med.data.model.ihis.nuro;

import java.util.Date;

public class NUR2016U02GrdListInfo {
	private String selectN;
	private String hospCodeLink;
	private String yoyangName;
	private String bunhoLink;
	private String suname;
	private String address1;
	private Date birth;
	public NUR2016U02GrdListInfo(String selectN, String hospCodeLink, String yoyangName, String bunhoLink,
			String suname, String address1, Date birth) {
		super();
		this.selectN = selectN;
		this.hospCodeLink = hospCodeLink;
		this.yoyangName = yoyangName;
		this.bunhoLink = bunhoLink;
		this.suname = suname;
		this.address1 = address1;
		this.birth = birth;
	}
	public String getSelectN() {
		return selectN;
	}
	public void setSelectN(String selectN) {
		this.selectN = selectN;
	}
	public String getHospCodeLink() {
		return hospCodeLink;
	}
	public void setHospCodeLink(String hospCodeLink) {
		this.hospCodeLink = hospCodeLink;
	}
	public String getYoyangName() {
		return yoyangName;
	}
	public void setYoyangName(String yoyangName) {
		this.yoyangName = yoyangName;
	}
	public String getBunhoLink() {
		return bunhoLink;
	}
	public void setBunhoLink(String bunhoLink) {
		this.bunhoLink = bunhoLink;
	}
	public String getSuname() {
		return suname;
	}
	public void setSuname(String suname) {
		this.suname = suname;
	}
	public String getAddress1() {
		return address1;
	}
	public void setAddress1(String address1) {
		this.address1 = address1;
	}
	public Date getBirth() {
		return birth;
	}
	public void setBirth(Date birth) {
		this.birth = birth;
	}
	

}
